src/tiledimage.js

```diff
@@ -1302,6 +1303,19 @@ function drawTiles( tiledImage, lastDrawn ) {
         return;
     }
     var useSketch = tiledImage.opacity < 1;
+    var sketchScale = 1;
+
+    var zoom = tiledImage.viewport.getZoom();
```

Comment:
You need to pass `true` into the getZoom; otherwise it's not actually accurate for the current frame (and instead just gives the zoom we're animating towards).